tests/page-objects/apdex/load.page.js

```diff
@@ -10,10 +10,15 @@ class LoadPage extends Page {
     return $('//android.widget.Button[@resource-id="org.medicmobile.webapp.mobile:id/btnSaveSettings"]');
   }
 
-  async loadInstance(url) {
+  async loadInstance(settingsProvider) {
+    const isServerSettings = await super.isButtonExisting('Custom');
+    if (!isServerSettings) {
```

Comment:
Skips instance setup. When the app is already configured, it won't show the instance setup page.